Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -307,6 +307,22 @@ - (void)sessionReplayStartedWithReplayId:(SentryId *)replayId
     return result;
 }
 
+- (nullable NSString *)currentScreenNameForSessionReplay
+{
+    __block NSString *screenName;
+
+    [SentrySDK.currentHub
+        configureScope:^(SentryScope *_Nonnull scope) { screenName = scope.currentScreen; }];
+
+    if (screenName == nil) {
+        screenName =
+            [SentryDependencyContainer.sharedInstance.application relevantViewControllersNames]
+                .firstObject;
+    }
+
+    return screenName;
```

Comment:
```suggestion
    return SentrySDK.currentHub.scope.currentScreen ?: [SentryDependencyContainer.sharedInstance.application relevantViewControllersNames].firstObject
```
I don't understand why this is using a block variable and `configureScope`, what am I missing here? Same goes for the preexisting `breadcrumbsForSessionReplay`.

Reply:
I'm so used to using configureScope to change something in a race-safe manner that I did it for reading too. 😆